mappings/net/minecraft/entity/mob/GuardianEntity.mapping

```diff
@@ -3,10 +3,10 @@ CLASS net/minecraft/class_1577 net/minecraft/entity/mob/GuardianEntity
 	FIELD field_7281 spikesExtensionRate F
 	FIELD field_7282 beamTicks I
 	FIELD field_7283 flopping Z
-	FIELD field_7284 prevSpikesExtension F
-	FIELD field_7285 tailAngle F
-	FIELD field_7286 spikesExtension F
-	FIELD field_7287 prevTailAngle F
+	FIELD field_7284 prevTailAngle F
+	FIELD field_7285 spikesExtension F
+	FIELD field_7286 tailAngle F
+	FIELD field_7287 prevSpikesExtension F
```

Comment:
Are these a result of matcher mismatch? Make sure these are applicable to 1.16

Reply:
After a quick investigation (and backporting manually mappings of `GuardianEntityModel` I have to say I'm a bit horrified when I saw that `GuardianEntity` has such a big mapping error in 1.16.5.

This backport is very applicable.